package.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/sourcegraph/sourcegraph/main/client/shared/src/schema/extension.schema.json",
   "name": "contact-author",
-  "description": "",
+  "description": "Contact the author (last committer) of any line of code, directly from a file view.",
```

Comment:
It seems a bit weird to say "contact the author" but then clarify that it will contact the _committer_. Wouldn't contacting the author make more sense? E.g. the person who wrote the PR, not the one who merged it.

EDIT: reading the code, it seems it does look at the author, not the committer, so it's just the description that is wrong.

Reply:
Maybe I'm conflating two separate concepts here. I assumed that is called "author" is the author of the last commit that touched that line but I may be missing the nuance of "author" in this context.

Reply:
> I assumed what is called "author" is the author of the last commit that touched that line

That statement is correct :)
But the description says "committer". "Committer" and "author" are two separate things in Git (and you can query both through our GraphQL API). See https://stackoverflow.com/questions/18750808/difference-between-author-and-committer-in-git